packages/cli-tools/CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Changed
+- Rename command `storage-node list-stream-parts` to `storage-node list-streams` and change the output format
```

Comment:
Just thought about this, it's more like the command "replaced" the old one than "renamed" since "rename" refers to the functionality staying the same (it's not since it now lists stream instead of stream parts).

```suggestion
- Replace command `storage-node list-stream-parts` with `storage-node list-streams` and change the output format
```